### PR TITLE
Refresh

### PR DESCRIPTION
**Pull request by fbiville**
As suggested by @jexp, I could also archive this repo and transfer [my fork](https://github.com/fbiville/neo4j-movies-go) to neo4j-examples.

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -46,168 +46,238 @@ type Link struct {
...
 }
 
-func main() {
-	serveMux := http.NewServeMux()
-	serveMux.HandleFunc("/", defaultHandler)
-	serveMux.HandleFunc("/search", searchHandler)
-	serveMux.HandleFunc("/movie/", movieHandler)
-	serveMux.HandleFunc("/graph", graphHandler)
+func parseConfiguration() *Neo4jConfiguration {
+	return &Neo4jConfiguration{
```
Comment (fbiville) [inline server.go:272]: I just need to parameterize this properly

Comment (fbiville) [reply server.go:272]: Or maybe not, [some examples](https://github.com/neo4j-examples/movies-javascript-bolt/blob/2e8324c7754cee3a53bc6bce5684892925594033/src/neo4jApi.js#L7) hardcode the password as well ^^

Comment (jexp) [reply server.go:272]: We should use env variables for the credentials and default perhaps to the demo.neo4jlabs.com one with movies/movies and db movies

**Comment by jexp** [comment]
Yes please archive this one and create a new repo from scratch.

**Pull request closed without merging**